ci/test_cudf_polars_polars_tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+# We will only fail these tests if the PR touches code in pylibcudf
+# or cudf_polars itself.
+# Note, the three dots mean we are doing diff between the merge-base
+# of upstream and HEAD. So this is asking, "does _this branch_ touch
+# files in cudf_polars/pylibcudf", rather than "are there changes
+# between upstream and this branch which touch cudf_polars/pylibcudf"
+# TODO: is the target branch exposed anywhere in an environment variable?
+if [ -n "$(git diff --name-only origin/branch-24.08...HEAD -- python/cudf_polars/ python/cudf/cudf/_lib/pylibcudf/)" ];
```

Comment:
should `libcudf` changes also trip this?

Reply:
I decided no for the previous round of running _our_ tests. Since polars might reasonably update their test suite independently of making breaking changes and introducing something that we don't support (and therefore would necessitate a new xfail) I would rather not use this to block libcudf-only merges.